service/app/controllers/coursemanagement/validator/CourseBatchRequestValidator.java

```diff
@@ -45,10 +45,19 @@ public void validateUpdateCourseBatchRequest(Request request) {
             ERROR_CODE);
       }
     }
-    validateParam(JsonKey.NAME, ResponseCode.mandatoryParamsMissing, JsonKey.NAME);
+    if (request.getRequest().containsKey(JsonKey.NAME)) {
```

Comment:
if it is mandatory parameter, we should be calling validateParam directly -  should not check for containsKey. This is only checking - if it contains - it should not be null